### ISSUE TITLE
Need two more branches legacy & post Mojave 

### ISSUE THREAD

**Issue by Gcenx** [vitor251093/wineskin#30]
As the title says

Gonna need a legacy branch for that 10.6.8 support & one for post Mojave. 

I already have a working branch for post Mojave with some caveats;

Still needs a 64bit replacement for 7zip for Wineskin/winery & wsgamma I believe is only 32bit so need a replacement for that.

That’s all Mojave beta complained about Wineskin/winery of course it still complains about WINE but that’s not really on us to implement.

**Comment by Gcenx** [comment]
Since my current pull requests are not merged yet you should be able to just make two branches from within Xcode using the current Head. Then upload them might even be able to do it from GitHub but I never looked I rather do it all from Xcode.

Also don't merge yet anyway I want to do some more work before we merge that into the master branch, I will just rebase one of my branches and open a new pull-request so its a cleaner 

I have not joined yet, once I get paid I can get my unlimited plan back and then I will properly just leave myself logged in.

@chrisballinger did you join the Porting Team Discord?

**Comment by vitor251093** [comment]
Just created the new branches :)
https://github.com/vitor251093/wineskin/tree/LegacySupport
https://github.com/vitor251093/wineskin/tree/PostMojave

**Comment by Gcenx** [comment]
Yeah I noticed but that's nothing they can easy be updated to also include the updated z7a

**Comment by chrisballinger** [comment]
@Gcenx I wasn't invited, where is it?

**Comment by Gcenx** [comment]
> @Gcenx I wasn’t invited, where is it

Oh you don’t use http://portingteam.com no wonder. Let you want me to send you a invite to the email listed on your github account?
